packages/workspace/src/utils/lint.ts

```diff
@@ -230,7 +230,7 @@ const globalESLint = `
   "parserOptions": {
     "ecmaVersion": 2018,
     "sourceType": "module",
-    "project": "./tsconfig.base.json"
+    "project": "./tsconfig.*.json"
```

Comment:
```suggestion
    "project": "./tsconfig.(base|lib).json"
```
might be a better way to approach this? that way we never pick up the tsconfig.spec.json

Reply:
No, I think spec files need to be linted as well. 

Nx 10+ typescript is configured to work as solution system. That means there would be a specific tsconfig file for each type of utility, and be scoped to the files mentioned in the respected `includes` or `files` property. 

For example, lib, app, e2e would have types that are needed for their own individual purpose. Spec configs would target `*.spec.ts` and lib/app configs target `!*.spec.ts`

Linting should work the same across all files, so we need to make sure we catch all configs.